manifests/mod/passenger.pp

```diff
@@ -1,5 +1,7 @@
 # @summary
 #   Installs `mod_pasenger`.
+#   > **Note**: This module does not install the passenger gem. You must install the gem yourself.
+#   > **Note**: This module support Passenger 5.0.0 and higher.
```

Comment:
EPEL 7 includes Passenger 4.0.53 now (https://src.fedoraproject.org/rpms/passenger).

Reply:
The current version of Passenger is 6, thats where I am thinking to keep it N-1 and N. 
I know I am going aggressive but open for thoughts, do you propose to keep it 4.0.0 as min version to support?

Reply:
Have updated with 4.0.0 as minimum version, open for thoughts.

Reply:
I'd indeed be a bit more conservative. Not sure how many users use it, but I tend to prefer distro packages. You can drop it next year when EPEL7 goes EOL.

Reply:
@ekohl Not sure if you got chance to look above comment but I have updated min passenger version to 4.0.0, I think that should be good?
Also the PR is ready for review, can you please review. Thanks

Reply:
Once everything looks good, I will release [11.0.0](https://github.com/puppetlabs/puppetlabs-apache/pull/2464), if we feel not good then will merge https://github.com/puppetlabs/puppetlabs-apache/pull/2469 with 11.0.0.

Reply:
I don't know the Puppet support policy (is there any?), but I would also be fine with only supporting what's in RHEL8/RHEL9. RHEL8 was released quite some time ago so one could argue that it's fine to Drop EL7. I'm fine with both.